.github/workflows/ci.yml

```diff
@@ -91,14 +133,86 @@ jobs:
       - name: Install oneAPI Compiler
         if: matrix.cxx_compiler == 'dpcpp' || matrix.cxx_compiler == 'icpx'
         run: sudo apt install -y intel-oneapi-compiler-dpcpp-cpp
+      - name: Install oneAPI Compiler (classic)
+        if: matrix.cxx_compiler == 'icpc'
+        run: sudo apt install -y intel-oneapi-compiler-dpcpp-cpp-and-cpp-classic
       - name: Run testing
         shell: bash
         run: |
           set -x
           sudo rm -rf /opt/intel/oneapi/dpl || true
           source /opt/intel/oneapi/setvars.sh intel64 || true
+          if [[ "${{ matrix.cxx_compiler }}" == "dpcpp" ]]; then
```

Comment:
Is there a universal way to print compiler version? If yes, just print it without any `if`

Reply:
There is no universal way on Windows